src/Components/BackOffice/BackOffice.jsx

```diff
@@ -62,7 +62,7 @@ function BackOffice() {
 
   return (
     <>
-      {token === false ? <Navigate to="/" /> :
+      {//token === false ? <Navigate to="/" /> :   lo comento porque me registra error token undefined
```

Comment:
como podrias modificar esta condicion entonces?

Reply:
En el merge del main, realizo este cambio para que no salte el mismo error:

!token ? <Navigate to="/" /> : token === false ? <Navigate to="/" /> :